reference/docs-conceptual/dev-cross-plat/create-standard-library-binary-Module.md

```diff
@@ -40,10 +40,36 @@ MyModule
 └───Tests
 ```
 
-## Getting Started
+## Why binary modules?
+
+When you are writing a module in C# you give up easy access to PowerShell cmdlets and functions. But
+if you are creating a module that doesn't depend on a lot of other PowerShell commands, the
+performance benefit can be significant. PowerShell was optimized for the administrator, not the
+computer. By switching to C#, you get to shed the overhead added by PowerShell.
+
+For example, we have a critical process that does a lot of work with JSON and hashtables. We
+optimized the PowerShell as much as we could but the process still takes 12 minutes to complete. The
+module already contained a lot of C# style PowerShell. This made conversion to a binary module clean
+and easy to do. The conversion cut reduced the process time from over 12 minutes to under four
+minutes.
```

Comment:
Since we moved this section from later in the article to before the changes, I think we might want to alter the wording here.

```suggestion
module already contained a lot of C# style PowerShell. This makes conversion to a binary module
clean and easy to do. The conversion in this article reduces the process time from over 12 minutes
to under four minutes.
```